node-js-client/scripts/tap-coverage.js

```diff
@@ -0,0 +1,38 @@
+#!/usr/bin/env node
+
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+/* eslint-disable no-console */
+if (!process.env.NO_COVERAGE) {
+  var ass = require('ass').enable( {
```

Comment:
Thanks @jrgm looks like fxa-auth-mailer is using nyc, so I'll check that out and get this updated.